ui-admin/src/portal/publish/PortalEnvDiffView.tsx

```diff
@@ -132,6 +132,13 @@ export default function PortalEnvDiffView(
           { !changeSet.siteContentChange.changed && <VersionChangeView record={changeSet.siteContentChange}/> }
         </div>
       </div>
+      <div className="my-2">
+        <h2 className="h6">
+          Dashboard config</h2>
+        <div className="ms-4 ">
+          <span className="text-muted fst-italic">not yet implemented</span>
```

Comment:
I still need to add this to the content publishing flow, but didn't want to hold up review for that.